lib/components/Expandable.tsx

```diff
@@ -7,9 +7,12 @@ export type ExpandableProps = PropsWithChildren<{
   label: ReactNode,
   icon?: (expanded: boolean) => ReactNode,
   initialExpansion?: boolean,
+  /**
+   * Whether the expansion should only happen when the header is clicked or on the entire component
+   */
+  clickOnlyOnHeader?: boolean,
```

Comment:
Whats the reason behind introducing this new prop?

Reply:
Normal `Expandable`s only expand when clicking the header as the content inside the expansion section might also use click events, and thus it would be confusing to have the expansion minimize on click. In other use cases like the FAQ Section on the other hand it would be confusing if the expansion only closes upon clicking the header.